cli/ops/process.rs

```diff
@@ -101,8 +103,28 @@ fn op_run(
     c.stderr(file);
   }
 
-  // We want to kill child when it's closed
-  c.kill_on_drop(true);
+  #[cfg(windows)]
+  {
+    let mut win_flags: u32 = 0x00000000;
+    if detached {
+      // Detach process on windows
+      win_flags = win_flags | 0x00000008;
```

Comment:
* Please use the name of this constant (`winapi::um::winbase::DETACHED_PROCESS`), don't hard code the value.
* If you want `detached` to have the same semantics as it has in node.js, you'll also need to add the `CREATE_NEW_PROCESS_GROUP` flag.

Reply:
Thanks a lot,

I currently do not have a windows box to test this with so it's a bit difficult for me to judge whether the implementation is working properly or not